doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -141,3 +141,11 @@ By default this Kconfig option is set to 1 second.
 .. note::
   The :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is set to ``y`` by default.
   To disable removing application-specific non-volatile data when the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` Kconfig option is selected, set the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option to ``n``.
+
+Read Client feature
+===================
+
+The Read Client feature is responsible for reading attributes from another device in the Matter network.
+The feature shall be activated when the device requires the capability to access attributes from a different device within the Matter network, for example, using bindings.
+This functionality is disabled by default for Matter samples in the |NCS|, except for ones that need to read attributes from the bound devices, such as the :ref:`matter_light_switch_sample` sample and the :ref:`matter_bridge_app` application.
```

Comment:
```suggestion
This functionality is disabled by default for Matter samples in the |NCS|, except for ones that need to read attributes from the bound devices, such as the :ref:`matter_light_switch_sample` sample and the :ref:`matter_bridge_app` application.
Enable the feature if your device needs to be able to access attributes from a different device within the Matter network to, for example, use bindings.
```
Not 100% sure if it should be "to, for example, use binding" as I suggested, or "using, for example, bindings".

Reply:
It should be "using, for example, bindings" :)